### PR TITLE
[ALLUXIO-2799] Use precondition check chaining in LineageInfo

### PR DESCRIPTION
**Pull request by nanshoudeooo**

### CODE REVIEW

**Comment by calvinjia** [comment]
alluxio-bot, check this please

File: core/common/src/main/java/alluxio/wire/LineageInfo.java [hunk 1]
```diff
@@ -103,6 +103,7 @@ public long getCreationTimeMs() {
   /**
    * @return the lineage children
    */
+
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/wire/LineageInfo.java]: Please remove the extra new line

File: core/common/src/main/java/alluxio/wire/LineageInfo.java [hunk 2]
```diff
@@ -122,7 +123,7 @@ public LineageInfo setId(long id) {
    */
   public LineageInfo setInputFiles(List<String> inputFiles) {
     Preconditions.checkNotNull(inputFiles);
-    mInputFiles = new ArrayList<>(inputFiles);
+    mInputFiles = new ArrayList<>(Preconditions.checkNotNull(inputFiles));
```
Comment (calvinjia) [inline core/common/src/main/java/alluxio/wire/LineageInfo.java]: Since we are adding the precondition here, the previous line can be removed.

**Comment by apc999** [comment]
@nanshoudeooo could you sign [the contribution license agreement](http://cla.alluxio.org/)?
also can you resolve the merge conflict?

**Comment by yupeng9** [comment]
@nanshoudeooo  feel free to reopen this PR after you sign the CLA

**Pull request closed without merging**